extensions/wikia/Search/classes/Result/ResultHelper.php

```diff
@@ -49,19 +52,37 @@ public static function extendResult($result, $pos, $descWordLimit)
 			: $result->getText(Utilities::field('description'), $descWordLimit);
 
 		$service = new MediaWikiService();
+		$wikiaSearchHelper = new \WikiaSearchHelper();
+
+		$globalSearchUrl = '';
+		if ( $query ) {
+			$lang = $wikiaSearchHelper->getLangForSearchResults();
+			$centralUrl = $wikiaSearchHelper->getCentralUrlFromGlobalTitle( $lang );
+			$globalSearchUrl = $wikiaSearchHelper->getGlobalSearchUrl( $centralUrl ) . '?search='. $query;
+		}
 
 		return [
 			'isOnWikiMatch' => isset($result['onWikiMatch']) && $result['onWikiMatch'],
 			'imageURL' => $imageURL,
 			'description' => $description,
-			'pagesMsg' => $service->shortnumForMsg($result['articles_i'] ?: 0, 'wikiasearch2-pages'),
-			'imgMsg' => $service->shortnumForMsg($result['images_i'] ?: 0, 'wikiasearch2-images'),
-			'videoMsg' => $service->shortnumForMsg($result['videos_i'] ?: 0, 'wikiasearch2-videos'),
+			'pages' => [
+				'count' => $service->shortNumForMsg($result['articles_i'] ?: 0),
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "count" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1192)